src/pages/Home.vue

```diff
@@ -30,6 +28,11 @@ export default {
       searchQuery: "",
       tableColumns: ["name", "no. of learners"],
       tableData: [],
+      // dummy table data - to show skeletons when data is being loaded
+      dummyTableData: Array(5).fill({
+        name: { type: "component", value: "dummy" },
+        "no. of learners": "-",
+      }),
```

Comment:
can we use the pending var or something else that already exists apart from creating a new dummy thing?